vnext/Microsoft.ReactNative/Modules/AppStateModule.cpp

```diff
@@ -20,6 +20,7 @@ void AppState::Initialize(winrt::Microsoft::ReactNative::ReactContext const &rea
       auto currentApp = xaml::TryGetCurrentApplication();
 
       if (!IsWinUI3Island() && currentApp != nullptr) {
+#ifndef USE_WINUI3
```

Comment:
we probably need something for WinAppSDK here, responding to [`WM_ACTIVATEAPP`](https://docs.microsoft.com/en-us/windows/win32/winmsg/wm-activateapp)